src/LSST/M1M3/SS/States/StandbyState.cpp

```diff
@@ -50,11 +53,25 @@ States::Type StandbyState::update(UpdateCommand* command) {
 States::Type StandbyState::start(StartCommand* command) {
     SPDLOG_INFO("StandbyState: start()");
     SettingReader::instance().getSafetyControllerSettings()->ForceController.exitBumpTesting();
+    RaisingLoweringInfo::instance().sendUpdates(true);
 
     Model::get().loadSettings(command->getConfigurationOverride());
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->version =
-            SettingReader::instance().getSettingsVersion();
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->otherInfo = "";
+
+    auto configurationApplied = M1M3SSPublisher::instance().getEventConfigurationApplied();
+    configurationApplied->configurations = "_init";
+    configurationApplied->version = GIT_HASH;
+    configurationApplied->url = CONFIG_URL;
```

Comment:
This needs to be extended to where the configuration lives within the storage location, in this case, the support system repo. It also includes all the way down to the schema directory. So, `CONFIG_URL` should be: `https://github.com/lsst-ts/ts_m1m3support/SettingFiles/v1`